dashboards/elasticsearch/elasticsearch.json

```diff
@@ -0,0 +1,2250 @@
+{
+  "__inputs": [],
```

Comment:
My only concern is that we define an input in all of our other dashboards here (e.g. https://github.com/dcos/grafana-dashboards/blob/master/dashboards/Edge-LB/Edge-LB-Apiserver_Apiserver.json#L3-L10) so I'd rather stick to convention. Do we anticipate this dashboard using a different datasource than the dcos-monitoring Prometheus? Otherwise, the only other thing that stood out to me was that maybe we should skip the `server` variable/filter and instead just pass in the datasource input (e.g. https://github.com/dcos/grafana-dashboards/blob/master/dashboards/Edge-LB/Edge-LB-Apiserver_Apiserver.json#L99).

Reply:
Thx! Updated.

Reply:
@gracedo Could you please take a look on the changes?